packages/webapp/src/views/layout.html

```diff
@@ -55,7 +55,7 @@
     homepageUrl: "/",
     containerClasses: "govuk-width-container",
     serviceName: serviceName,
-    serviceUrl: "/"
+    serviceUrl: serviceUrl
```

Comment:
`serviceUrl: serviceUrl or "/"` should give us a default that can be overriden by a `serviceUrl` if one is set

Reply:
updated